### ISSUE TITLE
[Bug]: Herd toolbar is opened when other apps are closed

### ISSUE THREAD

**Issue by siarheipashkevich** [beyondcode/herd-community#1183]
### Platform

macOS

### Operating system version

macOS Sequoia 15.1.1 (24B91)

### System architecture

Intel (x86)

### Herd Version

1.13.0 (Build: 37)

### Bug description

https://github.com/user-attachments/assets/2237fe2e-cd3d-4695-9615-3c5c7667c3ae

One more case:
https://github.com/user-attachments/assets/dc50e5ac-2fd7-4df0-bd92-ea2c4fea2492

### Steps to reproduce

The Herd just in the tray and when we are closing any app (in my example I'm closing Warp using cmnd + Q) the Herd toolbar is opened.

**Comment by mpociot** [comment]
Your video unfortunately does not show your Dock.
Is the Herd icon visible in the Dock and does it become "active" (as it's next in line) when you close a "previous" app?

**Comment by siarheipashkevich** [comment]
https://github.com/user-attachments/assets/d1d1e2f6-ba4a-438c-ad4f-247f02ad6507

@mpociot when I'm working in other apps my toolbar is not active, but after closing app where I worked Herd toolbar is opened and it's awful

**Comment by siarheipashkevich** [comment]
https://github.com/user-attachments/assets/619e05be-082e-4860-b930-54589d93e887

There is I'm showing that the last active icon of the toolbar is not Herd

**Comment by siarheipashkevich** [comment]
I'm using Herd a lot of time, but this issue happened after upgrading to 1.13.0 version

**Comment by mpociot** [comment]
Yeah, 1.13.0 introduced re-opening the toolbar when the app becomes active again and no windows are open. This should only happen when you re-open Herd via Spotlight/Raycast, etc.

This sounds like a macOS bug to me 🤔

**Comment by siarheipashkevich** [comment]
Hi @mpociot it's also valid for `ARM64 (M1)` system architecture.

**Comment by commeco** [comment]
Same here

**Comment by mpociot** [comment]
This was fixed a couple of versions ago

**Comment by siarheipashkevich** [comment]
@mpociot please merge https://github.com/mpociot/teamwork/pull/161
